src/dreyfus_index_updater.erl

```diff
@@ -160,6 +165,18 @@ update_task(NumChanges) ->
     couch_task_status:update([{progress, Progress}, {changes_done, Changes2}]).
 
 
+maybe_skip_doc(Db, Id) ->
```

Comment:
we can match the _design prefix in the function head which should make for a simpler function with less nesting;

```
maybe_skip_doc(Db, <<"_design/", _/binary">>) ->
    couch_db:is_partitioned(Db);
maybe_skip_doc(_Db, _Id) ->
    false.
```

Reply:
yes, make it simpler via 12ce31d